docs/resources/samples/combine-worksheets-into-single-workbook.md

```diff
@@ -15,7 +15,7 @@ This sample shows how to pull data from multiple workbooks into a single, centra
 ## Solution
 
 1. Create a new Excel file in your OneDrive. The file name "Combination.xlsx" is used in this sample.
-1. Create and save the two scripts from this sample.
+1. Create and save the two scripts from this sample as '.osts' files in your OneDrive folder under **Documents\Office Scripts**.
```

Comment:
Fair point that you can manually save the files. I envisioned folks saving the files with the Code Editor. How does this combined suggestion sound?
```suggestion
1. Create and save the two scripts from this sample. Either paste them into the Code Editor or create new '.osts' files in your OneDrive folder under **Documents\Office Scripts**.
```